cities/stuttgart.json

```diff
@@ -0,0 +1,447 @@
+{
+  "features": [
+    {
+      "geometry": {
+        "coordinates": [
+          48.7752013,
+          9.1784254
+        ],
+        "type": "Point"
+      },
+      "properties": {
+        "osm_id": "n4986263386",
+        "location": "Schillerplatz, Stuttgart-Mitte, Stuttgart",
+        "title": "Wochenmarkt Stuttgart Mitte / Schillerplatz",
+        "opening_hours": "Tu,Th 07:00-13:00, Sa 07:00-13:30"
+      },
+      "type": "Feature"
+    },
+    {
+      "geometry": {
+        "coordinates": [
+          48.77710,
+          9.17861
+        ],
+        "type": "Point"
+      },
+      "properties": {
+        "location": "Schillerplatz, Stuttgart-Mitte, Stuttgart",
+        "title": "Wochenmarkt Stuttgart Mitte / Schillerplatz",
+        "opening_hours": "Tu,Th 07:00-13:00, Sa 07:00-13:30"
+      },
+      "type": "Feature"
+    },
+    {
+      "geometry": {
+        "coordinates": [
+          48.7712849,
+          9.1783253
+        ],
+        "type": "Point"
+      },
+      "properties": {
+        "osm_id": "n2143176524",
+        "location": "Wilhelmsplatz, Stuttgart-Mitte, Stuttgart",
+        "title": "Wochenmarkt Stuttgart Mitte / Wilhelmsplatz",
+        "opening_hours": "Fr 12:00-18:00"
+      },
+      "type": "Feature"
+    },
+    {
+      "geometry": {
+        "coordinates": [
+          48.7839171,
+          9.2099166
+        ],
+        "type": "Point"
+      },
+      "properties": {
+        "osm_id": "n5246381963",
```

Comment:
Can you please explain what would be the benefit of introducing this new `osm_id` property? Is the identifier stable or does it change in the OSM database? I understand that you like to reference where you found the coordinates. However, at this point in time the `osm_id` is ignored by the _Wo ist Markt?_ website. Therefore, I suggest you move your _proposal_ (adding `osm_id`, `data_source_coords`) into a **separate GitHub** issue so we can **discuss** it there - and meanwhile unblock your pull request here.

Reply:
Agreed. I removed the additional properties with 07f94f4 and added OSM attribution verbally with 527ac7.

IMO, the id is more or less stable. In addition to being a reference, adding it would allow additional cross-reference checks, e.g. for completeness of one or the other, comparing opening_hours etc. 

Concerning data_source_coords, I'd rather propose using an array, so multiple sources may be declared. 

I'll create feature requests for both.

Reply:
Follow up discussions: https://github.com/wo-ist-markt/wo-ist-markt.github.io/issues/287 and https://github.com/wo-ist-markt/wo-ist-markt.github.io/issues/288.